Drop-In/src/androidTest/java/com/braintreepayments/api/SupportedPaymentMethodsFragmentUITest.kt

```diff
@@ -0,0 +1,259 @@
+
+        val scenario = FragmentScenario.launchInContainer(SupportedPaymentMethodsFragment::class.java, bundle)
+        scenario.moveToState(Lifecycle.State.RESUMED)
+
+        scenario.onFragment { fragment ->
+            fragment.dropInViewModel.setSupportedPaymentMethods(supportedPaymentMethods)
+        }
+
+        onView(isRoot()).perform(waitFor(500))
+        onView(withText("Credit or Debit Card")).perform(click())
+        scenario.onFragment { fragment ->
+            val activity = fragment.requireActivity()
+            val fragmentManager = fragment.parentFragmentManager
+            fragmentManager.setFragmentResultListener(DropInEvent.REQUEST_KEY, activity) { _, result ->
+                val event = DropInEvent.fromBundle(result)
+                assertEquals(DropInEventType.SUPPORTED_PAYMENT_METHOD_SELECTED, event.type)
+
+                val paymentMethodType =
+                    event.getDropInPaymentMethodType(DropInEventProperty.SUPPORTED_PAYMENT_METHOD)
+                assertEquals(DropInPaymentMethodType.UNKNOWN, paymentMethodType)
+                countDownLatch.countDown()
+            }
+        }
+
+        countDownLatch.await()
+    }
+
+    @Test(timeout = 5000)
+    fun whenStateIsRESUMED_whenVaultManagerEnabledAndVaultedCardExists_sendsAnalyticEvent() {
+        // TODO: capture all analytics events within a time interval and assert that the target analytics event is emitted
+
+//        val dropInRequest = DropInRequest()
+//                .vaultManager(true)
+//        val bundle = bundleOf("EXTRA_DROP_IN_REQUEST" to dropInRequest)
+//
+//        val scenario = FragmentScenario.launchInContainer(SelectPaymentMethodFragment::class.java, bundle)
+//        scenario.moveToState(Lifecycle.State.RESUMED)
+//
+//        scenario.onFragment { fragment ->
+//            val cardNonce = CardNonce.fromJSON(JSONObject(Fixtures.PAYMENT_METHODS_VISA_CREDIT_CARD))
+//            val vaultedPaymentMethods = listOf(cardNonce)
+//
+//            fragment.dropInViewModel.setSupportedPaymentMethods(supportedPaymentMethods)
+//            fragment.dropInViewModel.setVaultedPaymentMethods(vaultedPaymentMethods)
+//
+//            val activity = fragment.requireActivity()
+//            val fragmentManager = fragment.parentFragmentManager
+//
+//            fragmentManager.setFragmentResultListener(DropInEvent.REQUEST_KEY, activity) { requestKey, result ->
+//                val event = result.get(DropInEvent.RESULT_KEY) as DropInAnalyticsEvent
+//                assertEquals(event.fragment, "vaulted-card.appear")
+//                countDownLatch.countDown()
+//            }
+//        }
+//        countDownLatch.await()
+    }
+
+    @Test
+    fun whenStateIsRESUMED_whenVaultManagerEnabledAndVaultedCardDoesNotExist_doesNotSendAnalyticEvent() {
+        // TODO: capture all analytics events within a time interval and assert that the target analytics event is emitted
```

Comment:
Same with these two tests, are we going to add functionality here or can we just remove them.